usaspending_api/search/migrations/0003_delete_universaltransactionview.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 2.2.14 on 2021-01-05 19:00
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('search', '0002_auto_20210104_2314'),
+    ]
+
+    operations = [
```

Comment:
Is there an operational reason this is a separate migration?

Reply:
I was thinking it would be a good idea to do these steps:
1. Run first migration (create new table)
2. Run matview_runner with new code to populate new table
3. Deploy codebase
4. Run second migration to remove old table

Reply:
Since the model was unmanaged by Django, I would be surprised if the table would be dropped by the migration

Reply:
I just tested this, and you are correct. I will include these changes in a single migration